test/lib/token.test.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const assert = require('chai').assert;
+const sinon = require('sinon');
+const mockery = require('mockery');
+const schema = require('screwdriver-data-schema');
+
+sinon.assert.expose(assert, { prefix: '' });
+
+describe('Token Model', () => {
+    const password = 'password';
+    let BaseModel;
+    let TokenModel;
+    let ironMock;
+    let datastore;
+    let createConfig;
+    let token;
+
+    before(() => {
+        mockery.enable({
+            useCleanCache: true,
+            warnOnUnregistered: false
+        });
+    });
+
+    beforeEach(() => {
+        datastore = {
+            update: sinon.stub()
+        };
+        datastore.update.resolves({});
+
+        ironMock = {
+            seal: sinon.stub(),
+            unseal: sinon.stub(),
+            defaults: {}
+        };
+        mockery.registerMock('iron', ironMock);
+
+        // eslint-disable-next-line no-global-assign
+        Date = function Date() {
```

Comment:
That makes a lot of sense, I hadn't considered the effect that could have on other code. 

In this case I just want to check that timestamps actually get updated when they're supposed to, so rather than asserting that the value that comes back is a specific date I'll try storing the initial value, and then comparing the one that comes back against that to make sure it's changed.